sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -184,8 +184,11 @@ def test_hidden_output_file_workflow(self):
     """
     Test compiling a workflow with non configurable output file.
     """
-    from .testdata.hidden_output_file import hidden_output_file_pipeline
-    self._test_pipeline_workflow(hidden_output_file_pipeline, 'hidden_output_file.yaml')
+    if sys.version_info < (3, 6, 0):
+      logging.warning("Skipping hidden_output workflow test for Python version < 3.6.0")
```

Comment:
could we add a one-line comment about why we require Python `3.6.1` or above?

Reply:
Thanks @ckadner, I added a comment for it. It was due to some sorting function in the python 3.5 testing when we have certain combination of special characters.
https://travis-ci.com/github/kubeflow/kfp-tekton/jobs/384130977